project-metrics/metrics_service/server.py

```diff
@@ -28,5 +28,16 @@ def recompute(metric_cls_name):
   return 'Successfully recomputed %s.' % metric_cls_name, status.HTTP_200_OK
 
 
+@app.route('/api/metrics')
+def list_metrics():
+  try:
+    results = base.Metric.get_latest()
+  except Exception as error:
```

Comment:
Can you narrow this to a more specific exception/list of exceptions? (`IOError` comes to mind)

Reply:
I want to catch any error and return valid JSON for the client to handle